docs/src/main/tut/docs/non-automatic-derivation.md

```diff
@@ -89,13 +90,7 @@ val conf = ConfigFactory.parseString("{ name: John, surname: Doe }")
 ```tut:silent
 import pureconfig._
 
-implicit val personReader = ConfigReader.fromCursor[Person] { cur =>
-  for {
-    objCur <- cur.asObjectCursor
-    name <- objCur.atKey("name").right.flatMap(_.asString)
-    surname <- objCur.atKey("surname").right.flatMap(_.asString)
-  } yield Person(name, surname)
-}
+implicit val personReader = ConfigReader.forProduct2("name", "surname")((name, surname) => Person(name, surname))
```

Comment:
Can you either use the shorthand `Person(_, _)` for the second argument or split this line into two?

Reply:
I've simplified the definition of `personReader` in e75495a.